scaleway/resource_instance_security_group.go

```diff
@@ -126,16 +137,25 @@ func resourceScalewayInstanceSecurityGroupRead(d *schema.ResourceData, m interfa
 	_ = d.Set("inbound_default_policy", res.SecurityGroup.InboundDefaultPolicy.String())
 	_ = d.Set("outbound_default_policy", res.SecurityGroup.OutboundDefaultPolicy.String())
 
-	//
-	// Handle SecurityGroupRules
-	//
+	if d.Get("external_rules") != nil && d.Get("external_rules").(bool) {
+		stateRules, err := getSecurityGroupRules(instanceApi, zone, ID, d)
```

Comment:
should be `inboundRules, outboundRules, err := getSecurityGroupRules(instanceApi, zone, ID, d)`